### PR TITLE
feat: add attribution position property

### PR DESCRIPTION
**Pull request by doomsower**
This PR adds `attributionPosition` property which allows to move attribution button to some corner other than default.

Motivation: I have FAB in bottom right corner and it doesn't play well with attribution button on iOS. Android added for parity.

### CODE REVIEW

**Comment by mattijsf** [comment]
I think this is a nice addition 👍

However, eventually I believe it better to create something like `<MapboxGL.AttributionView />` that makes use of [showAttribution](https://github.com/react-native-mapbox-gl/maps/blob/master/docs/MapView.md#showattribution) that can be laid-out on top of the map. Once we have that you can position it anywhere and even apply some animations / transitions.

Same idea can then be applied to `<MapboxGL.CompassView />`...

File: docs/MapView.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
...
+| attributionPosition | `object` | `none` | `false` | Add attribution offset, e.g. `{top: 8, left: 8}` will put attribution button in top-left corner of the map |
```
Comment (mfazekas) [inline docs/MapView.md]: pls add this to the propTypes, docs are generated

**Comment by doomsower** [comment]
I've added propTypes, but `generate` task fails for me due to missing `documentation` command. Is it generated somewhere on CI?

As for dedicated component: this is quite a lot of additional work. Would you merge this PR as it is?

Maybe I can find some time later to implement it. Do you think it can later be done in pure JS based on this prop and other existing native view props and methods?

**Comment by kristfal** [comment]
@doomsower:

I've updated CONTRIBUTING.md with answer to your question:

```
## Documentation

1. Documentation is auto-generated from code blocks and comments. To generate documentation:
  1. `npm install -g documentation` to install documentation generator
  2. `npm run generate` to generate updated documentation
```

**Comment by kristfal** [comment]
Merged! Thanks for your contributions

**Pull request merged**